dss-collection-nft/contracts/DSSCollection.cdc

```diff
@@ -0,0 +1,695 @@
+/*
+    DSSCollection contains collection group & completion functionality for DSS.
+    Author: Jeremy Ahrens jer.ahrens@dapperlabs.com
+*/
+
+import NonFungibleToken from "./NonFungibleToken.cdc"
+import MetadataViews from 0xMETADATAVIEWSADDRESS
+
+// The DSSCollection contract
+//
+pub contract DSSCollection: NonFungibleToken {
+
+    // Contract Events
+    //
+    pub event ContractInitialized()
+
+    // NFT Collection Events
+    //
+    pub event Withdraw(id: UInt64, from: Address?)
+    pub event Deposit(id: UInt64, to: Address?)
+
+    // Events
+    //
+    pub event CollectionGroupCreated(
+        id: UInt64,
+        name: String,
+        description: String,
+        typeName: String,
+        startTime: UFix64?,
+        endTime: UFix64?,
+        timeBound: Bool
+    )
+    pub event CollectionGroupClosed(id: UInt64)
+    pub event ItemCreated(
+        id: UInt64,
+        itemID: UInt64,
+        points: UInt64,
+        itemType: String
+    )
```

Comment:
Per a private slack discussion, we'll only let an item be in one slot.  It'll emit the fields below and we won't have `ItemAddedToSlot` anymore.  Thought I like this contract better the way it is from a purist standpoint (re-usable items), it's a lot harder to add to core currently :(.  This won't remove functionality as items can be identical to eachother.

```
 slotID: UInt64,
collectionGroupID: UInt64
```

Reply:
Have removed createItem in favor of createItemInSlot.